repo/content/info.go

```diff
@@ -13,11 +13,8 @@ type (
 	// IDPrefix represents a content ID prefix (empty string or single character between 'g' and 'z').
 	IDPrefix = index.IDPrefix
 
-	// Info is an information about a single piece of content managed by Manager.
-	Info = index.Info
-
-	// InfoStruct is an implementation of Info based on a structure.
-	InfoStruct = index.InfoStruct
+	// Info is an implementation of Info based on a structure.
```

Comment:
This comment is misleading since this is a type alias and there is no longer a `Info` interface, thus `content.Info` is not implementing an `Info` interface.

Reply:
reworded